operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -475,91 +481,107 @@ private void logThrowable(Throwable throwable) {
   }
 
   /**
-   * Begin activity to align domain status with domain resource.
-   * @param info domain presence info
-   * @param explicitRecheck if explicit recheck
+   * Begin activity to align the cached domain status with the value reaa from Kubernetes.
+   * @param liveInfo domain presence info read from Kubernetes
+   * @param explicitRecheck true if we are to skip comparing the reported domain to the cached value
    * @param isDeleting if is deleting domain
    * @param isWillInterrupt if will interrupt already running activities
    */
   public void makeRightDomainPresence(
-      DomainPresenceInfo info,
+      DomainPresenceInfo liveInfo,
```

Comment:
should we mark `liveInfo` `@Nonnull` too?

Reply:
Good idea, thanks.